### PR TITLE
readme update on how to add a new Blog Post

### PR DESCRIPTION
**Pull request by saadamir3180**
how to add new blog post to the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ _Agency Website for TECVITY_
   - [Email Configuration](#email-configuration)
   - [Database](#database)
   - [Disabled Features](#disabled-features)
+  - [Blog Post Contriution Guide](#blog-post-contribution-guide)
+  - [Service Addition Guide](#service-addition-guide)
```
Comment (njmulsqb) [inline README.md:21]: why is this added? only add when the section we're mentioning is also added

File: README.md [hunk 2]
```diff
@@ -118,8 +120,177 @@ Following feautres/pages exist in the codebase but have been disabled for now (c
...
+ ```
+
+### 9. Add the Author's Name
+Example:
+```javascript
+author: 'Author 1', 
+```
+
+### 10. Add the Author's Quote
+Example:
```
Comment (njmulsqb) [inline README.md:208]: Explain what is this, why is it required, and if it is an optional field (add this for rest of the values as well to explain if some field is optional)

File: README.md (hunk 1 above)
Comment (saadamir3180) [reply README.md:21]: added by mistake, removing in the next commit.

File: README.md [hunk 3]
```diff
@@ -118,8 +120,177 @@ Following feautres/pages exist in the codebase but have been disabled for now (c
...
+### 13. Add Blog Content
+Add the introduction and sections:
+```javascript
+ content: { 
+    introduction: 'We believe that the key to our success is our team. ...', 
+    sections: [ { heading: 'Heading', content: 'Some content' } ] 
+  } 
+  ```
+
+### 14. Add Blog Sections
```
Comment (njmulsqb) [inline README.md:234]: What is the difference between blog section and blog content? #14 and #15 ? 

Can we bold the text or apply some other formatting in blog content, can we do that?
Also what if we want to attach some images in the blog content?

**Review by njmulsqb — COMMENTED** [review]
Here's the review.

File: README.md [hunk 4]
```diff
@@ -118,8 +120,177 @@ Following feautres/pages exist in the codebase but have been disabled for now (c
...
+3. [Final Example BlogPost Object](#final-example-blogpost-object)
+
+---
+
+## Steps for Adding a New Blog Post
+
+### 1. Add an id for the Blog Post
+Assign an id that is one value greater than the previous blog post. If the previous Blog id is `1` use `2` Example:
+```javascript 
+id: 2,
```
[the remark this answers is not in the document]
Comment (saadamir3180) [reply README.md:156]: The comments that are posted are saved into the database based on a unique `id`. If this `id` collides, we cannot track which comment belongs to which blog post

File: README.md (hunk 2 above)
Comment (saadamir3180) [reply README.md:208]: Okay

File: README.md (hunk 3 above)
Comment (saadamir3180) [reply README.md:234]: Currently we can only apply list formatting in blog content, and blog currently supports two images one that is displayed on the preview card and one on the actual blog post.

**Review by njmulsqb — APPROVED** [review]
